src/transactions/InvokeHostFunctionOpFrame.cpp

```diff
@@ -79,10 +70,11 @@ getLedgerInfo(AbstractLedgerTxn& ltx, Config const& cfg,
     info.cpu_cost_params = toCxxBuf(sorobanConfig.cpuCostParams());
     info.mem_cost_params = toCxxBuf(sorobanConfig.memCostParams());
     // TODO: move network id to config to not recompute hash
-    auto networkID = sha256(cfg.NETWORK_PASSPHRASE);
```

Comment:
Nit: The comment above this ("// TODO: move network id to config to not recompute hash") can be removed.

Reply:
fixed